config/routes.rb

```diff
@@ -301,7 +301,7 @@
       end
 
       resources :questions, shallow: true do
-        resources :answers, except: [:index, :destroy, :delete], controller: 'questions/answers', shallow: true do
+        resources :answers, except: [:index, :destroy], controller: 'questions/answers', shallow: true do
```

Comment:
I don't think we need the `shallow: true` here if we have it declared on `resources :questions`

Reply:
👍